torch_xla/csrc/ops/dynamic_ir.cpp

```diff
@@ -118,6 +118,28 @@ int64_t SizeEq::getDynamicValue() const {
 
 std::string SizeEq::ToString() const { return "aten::eq_size"; }
 
+SizeGe::SizeGe(torch::lazy::Value a, torch::lazy::Value b)
```

Comment:
Is it possible to add some python unit test like https://github.com/pytorch/xla/blob/a4d48cb8717634c5991ca3135148db699706273f/test/test_dynamic_shapes.py#L63? I think when you do something like `t2.shape[0] >= t2.shape[1]` should trigger this SizeGe.

Reply:
Yea, it should be possible. On the other hands, do we have a way to validate if we indeed trigger SizeGe?